tutorials/axios/axios-patch.md

```diff
@@ -0,0 +1,13 @@
+The easiest way to make a PATCH request with [Axios](https://www.npmjs.com/package/axios) is the axios.patch() function. The first parameter is the url to which the request will be made, and the second parameter is the data you will be sending to change. You can include the other data that will not be changed as this will have no effect on the data stored. You can make a Patch request with axios as follows:
+
+```javascript
+[require:axios Patch patch]
+```
+
+If you omit the second parameter, axios will treat it like an object and do a JSON.stringify on the empty object before sending the request. It also will specify the content-type to [application/json](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type), allowing for smooth integration in most projects.
```

Comment:
I'm not sure what you're trying to say in these sentences. I think you're trying to explain that, if the 2nd parameter is an object, Axios will `JSON.stringify()` it and set the content-type to application/json? Also, I think there should be an example for this one.